libbeat/docs/dashboards.asciidoc

```diff
@@ -125,7 +125,7 @@ ifdef::allplatforms[]
 
 ["source","sh",subs="attributes"]
 ----
-docker run {dockerimage} setup -e \
+docker run --net="host" {dockerimage} setup -e \
   -E output.logstash.enabled=false \
   -E output.elasticsearch.hosts=['localhost:9200'] \
   -E output.elasticsearch.username={beat_default_index_prefix}_internal \
```

Comment:
Thanks for this change, indeed I don't think this can work without `--net=host`.